frontend/src/pages/category-page.tsx

```diff
@@ -262,35 +263,43 @@ const CategoryPageContent: React.FC<CategoryPageContentProps> = ({
           </Card>
 
           <Grid my="sm" gutter={{ base: "sm", sm: "md" }}>
-            <Grid.Col span={{ base: 12, md: 8 }}>
-              <Paper withBorder p={{ base: "sm", sm: "md" }}>
-                <ExamList metaData={metaData} />
+            {tabs.currentTabId === "statistics" ? (
+              <Grid.Col span={12}>
```

Comment:
What would be your opinion if I said I would prefer the right sidebar to stay across all tabs?

![Screen Recording 2025-09-28 at 05 20 54 (1)](https://github.com/user-attachments/assets/955e9b52-723b-4a22-8e7a-b376f882beef)

I was thinking about this, particularly with regards to course testimonials, where seeing the current exam/coursework split etc could be useful for putting the testimonials into context. Then for consistency, grades would also need such a treatment. The graph doesn't look particularly bad when squished tbh, since the data-ink ratio isn't very high to begin with.

(Although, we could change this post-merging, so it's not something I need you to change right now).